### PR TITLE
Soften test deps HTTP::Tiny::Mech and WWW::Mechanize::Cached

### PR DESCRIPTION
**Pull request by pghmcfc**
They are already listed as test recommends

### CODE REVIEW

**Comment by mickeyn** [comment]
thanks

File: cpanfile [hunk 1]
```diff
@@ -16,14 +16,12 @@ requires "warnings" => "0";
 
 on 'test' => sub {
   requires "File::Spec" => "0";
-  requires "HTTP::Tiny::Mech" => "0";
```
Comment (oalders) [inline cpanfile]: Could this just have been switched from `requires` to `recommends`?

**Comment by pghmcfc** [comment]
@oalders, it was both a `requires` and a `recommends`; I just got rid of the `requires`, leaving the `recommends` untouched.

**Comment by mickeyn** [comment]
@oalders the `recommends` already exist in a the separate section you added for it :)

**Comment by oalders** [comment]
I guess I shouldn't have checked *only* the diff.  :)  Thanks @pghmcfc and @mickeyn.

**Comment by mickeyn** [comment]
I think these were added by `dzil` automatically... i'll fix the `dist.ini` not to do that on every `build`.

**Comment by pghmcfc** [comment]
Something went wrong with the 1.028000 release; the deps are unchanged and cpanfile wasn't included in the release tarball.

**Comment by mickeyn** [comment]
yeah, my change did that.
the problem is that with the current setup dzil changes the cpanfile to include those lines you removed and they get included in the shipped cpanfile.
I'm trying to find a way to prevent that and will appreciate help if you are proficient with dzil.

**Comment by pghmcfc** [comment]
I'm afraid my skillset is in downstream packaging for Fedora rather than dzil; if I knew dzil I'd probably have been able to include whatever was needed in the PR ... sorry

**Comment by oalders** [comment]
See #51

**Comment by oalders** [comment]
Actually, I don't think that fixes this issue, but it fixes a build problem.

**Comment by oalders** [comment]
See #52 for the fix to this issue.

**Comment by mickeyn** [comment]
@oalders thanks for all the fixes. release 1.028001 will ship with a better cpanfile now.

**Comment by pghmcfc** [comment]
1.028001 now has the right cpanfile but the Makefile.PL and META.* still have the old hard dependencies.

**Comment by mickeyn** [comment]
@pghmcfc 1.028003 deals with that issue

**Comment by mickeyn** [comment]
thanks for reporting

**Pull request merged**